modules/table/src/test/java/org/apache/ignite/internal/table/distributed/AbstractMvStorageUpdateHandlerTest.java

```diff
@@ -95,7 +96,8 @@ void setUp(
                 PARTITION_ID,
                 partitionDataStorage,
                 DummyInternalTableImpl.createTableIndexStoragesSupplier(Map.of()),
-                distributionZoneConfig.dataStorage()
+                distributionZoneConfig.dataStorage(),
+                new PendingComparableValuesTracker<>(HybridTimestamp.MAX_VALUE)
```

Comment:
MAX_VALUE?
Interesting

Reply:
So that when inserting into a table, the GC always worked for us.